QuoteOfTheDay/Pages/Index.cshtml.cs

```diff
@@ -40,13 +40,11 @@ public async void OnGet()
 
         if (variant != null)
         {
-            ShowGreeting = variant.Configuration.Get<bool>();
+            Greeting = variant.Configuration?.Get<string>() ?? "";
         }
         else
         {
-            _logger.LogWarning("Greeting variant not found. Please define a variant feature flag in Azure App Configuration named 'Greeting' with 'true' and 'false' variants.");
-
-            ShowGreeting = false;
+            _logger.LogWarning("Greeting variant not found. Please define a variant feature flag in Azure App Configuration named 'Greeting'.");
```

Comment:
Can you define "Greeting" as a local const string that way it can be shared here and in `GetVariantAsync`?